Makefile

```diff
@@ -347,17 +351,23 @@ endif
 
 define RELEASE_SKIP_TESTS_HELP_INFO
 # Build a release, but skip tests
+# 
+# Args:
+#   KUBE_RELEASE_RUN_TESTS: Whether to run tests. Set to 'y' to run tests anyways.
+#   KUBE_FASTBUILD: Whether to cross-compile for other architectures. Set to 'true' to do so.
 #
 # Example:
 #   make release-skip-tests
+#   make quick-release
 endef
 .PHONY: release-skip-tests quick-release
 ifeq ($(PRINT_HELP),y)
 release-skip-tests quick-release:
 	@echo "$$RELEASE_SKIP_TESTS_HELP_INFO"
 else
+release-skip-tests quick-release: KUBE_RELEASE_RUN_TESTS = n KUBE_FASTBUILD = true
```

Comment:
this appears to be setting `KUBE_RELEASE_RUN_TESTS` to `n KUBE_FASTBUILD = true` and not affecting `KUBE_FASTBUILD`. `make` is the worst.

cc @mikedanese